spec/spec_helper.rb

```diff
@@ -54,12 +53,26 @@
 
   config.after :each do
     DatabaseCleaner.clean
-    delete_test_indices
   end
 
   config.after :suite do
     DatabaseCleaner.clean_with :truncation
-    delete_test_indices
+  end
+
+  config.before :each, bookmark_search: true do
```

Comment:
Should we have an after each counterpart for deleting the indices, in case tests not marked as using search try to use search?

Reply:
It shouldn't matter unless we're testing search-specific behavior, and if we are, we should be using one of the search tags. Plus, not much is going to happen without the reindexer task being invoked. I'd kind of like to err on the side of reducing redundant calls and see if we have any issues with that going forward.

Reply:
I suspect the issues would arise from people new to the codebase, who might be unfamiliar with which functions require ES. They might find it easier to debug their new tests (and realize that they're unintentionally using ES) if they get an error about the indices not existing, instead of blank or incorrect results.

Reply:
+1 to not leaking state between tests.